### ISSUE TITLE
[Search filter] Headings should be persistent on both mobile and desktop breakpints

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/vets-design-system-documentation#4166]
## Description

Update code to maintain heading structure in the mobile view of the  [search filter component](https://design.va.gov/storybook/?path=/docs/components-va-search-filter--docs)

## Details
- **Must**:  Semantic heading code should remain persistent on all breakpoints.  See WCAG Success Criteria: [WCAG 1.3.4 Orientation - Level AA](https://www.w3.org/WAI/WCAG22/Understanding/orientation); [WCAG 1.3.1 Info and relationships](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html) "Heading markup will allow assistive technologies to present the heading status of text to a user. A screen reader can recognize the code and announce the text as a heading with its level, beep or provide some other auditory indicator. "
- **Recommendation** When the accordions disappear from the mobile view, maintain the heading structure of the h3s in the legends.

## Tasks
- [ ] Update mobile breakpoint to maintain semantic heading structure

## Acceptance Criteria
- [ ] Heading structure follows as such on all breakpoints:
```
<h2>Filters</h2>
<h3>Benefits</h3>
<h3>Service Status</h3>
```
